src/main.cpp

```diff
@@ -16,18 +22,84 @@ class Command : public TgBot::BotCommand {
     }
 };
 
+bool HandleUser(size_t id, std::unordered_map<size_t, size_t>& users) {
```

Comment:
Not so obvious that `id` is a user id, a have been thinking for a 3 seconds to got it.

`users` I guess is an amount of messages be user id.

It is better to give it a more detailed name.

Reply:
done